code/client/src/components/SiderMenu.jsx

```diff
@@ -27,16 +32,27 @@ const SiderMenu = ({ ...args }) => {
   const nav = ({ key }) => {
     history.push(Paths[key])
   }
+  const [stats, setStats] = useState(null)
+
+  useEffect(() => {
+    async function getStats () {
+      const statsData = await util.getWalletStats()
+      setStats(statsData)
+    }
+    getStats()
+  }, [])
+
   return (
     <Layout.Sider collapsible={width < 900} onCollapse={c => setCollapsed(c)} {...args}>
       {/* <Image src='/assets/harmony.svg' /> */}
       <Row justify='center'>
-        <Link target='_blank' rel='noreferrer' href='https://harmony.one/'>
+        <Link target='_blank' rel='noopener noreferrer' href='https://harmony.one/'>
           <Image preview={false} src={collapsed ? HarmonyIcon : HarmonyLogo} style={{ cursor: 'pointer', padding: collapsed ? 16 : 32 }} onClick={() => history.push('/')} />
         </Link>
       </Row>
-      {!collapsed && <Text><Link href='https://harmony.one/1wallet' target='_blank' rel='noreferrer'>{config.appName} {config.version}</Link></Text>}
-      {!collapsed && <Row justify='center' style={{ marginBottom: 10 }}><Tag color='#0094c0'>Beta</Tag></Row>}
+      {!collapsed && <Text><Link href='https://harmony.one/1wallet' target='_blank' rel='noopener noreferrer'>{config.appName} {config.version}</Link></Text>}
+      <Row justify='center'><Stats>{stats && stats.count.toLocaleString()} wallets</Stats></Row>
```

Comment:
This needs to be fail safe. If stats is null (for example when API server is down) this will show up as "undefined wallets"